src/figuresClasses.ts

```diff
@@ -1,19 +1,109 @@
 export interface Figure {
-
+  shape: 'triangle' | 'rectangle' | 'circle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: 'triangle' = 'triangle';
 
-}
+  color: 'red' | 'green' | 'blue';
+
+  sideA: number;
+
+  sideB: number;
+
+  sideC: number;
+
+  constructor(
+    color: 'red' | 'green' | 'blue',
+    sideA: number,
+    sideB: number,
+    sideC: number,
```

Comment:
you can access Figure types with this syntax: `Figure['typeName']` - you won't have to retype everything again